python/setup.py

```diff
@@ -8,7 +8,7 @@
 import shutil
 
 _major = '0.3'
```

Comment:
I still think this should be major and minor

Reply:
done!